tests/store/test_cli.py

```diff
@@ -26,6 +27,46 @@ def test_file_info_to_json():
     }]
 
 
+def test_download_from_uri():
+    class TestArtifactRepo:
+        def __init__(self, scheme):
+            self.scheme = scheme
+
+        def download_artifacts(self, artifact_path, dst_path=None):
+            return (self.scheme, artifact_path)
+
+    def test_get_artifact_repository(artifact_uri, **kwargs):
+        return TestArtifactRepo(artifact_uri)
+
+    pairs = [
+        ("path", ("", "path")),
+        ("path/", ("path", "")),
+        ("/path", ("/", "path")),
+        ("/path/", ("/path", "")),
+        ("path/to/dir", ("path/to", "dir")),
+        ("file:///", ("file:///", "")),
```

Comment:
Nit: Can we test `file:` URIs with a single forward slash? E.g., `file:/path/to/artifact`.

Reply:
Is that a valid file uri?

Reply:
NEvermind, I looked it up and it is a valid path. I'll add a test case for it.